.aegir.js

```diff
@@ -45,7 +45,7 @@ const after = async () => {
 }
 
 module.exports = {
-  bundlesize: { maxSize: '179kB' },
+  bundlesize: { maxSize: '185kB' },
```

Comment:
This was updated meanwhile in master from 220 to 179, which had a minimal interval. I rebased `0.28.x` with master and this was immediately an issue, so I bumped it for now. We can analyse later on what the ideal value is